flexeval/core/language_model/litellm_api.py

```diff
@@ -28,7 +28,7 @@ class LiteLLMChatAPI(OpenAIChatAPI):
         string_processors: A single or a list of StringProcessor objects to process the model's output.
         ignore_seed: If True, ignore the seed specified in default_gen_kwargs.
             This is an option for models that do not support seed parameters such as anthropic/claude.
-        model_limit_new_tokens: An upper limit on the number of tokens the model can generate.
+        model_limit_completion_tokens: An upper limit on the number of tokens the model can generate.
```

Comment:
is this change intended?

Reply:
We might want to rebase? I am not sure tho…

Reply:
Oh, it is a quick fix of typo.
It is confusing but no problem.

Reply:
Cool, thanks! Let's merge then.